packages/at_onboarding_cli/lib/src/cli/auth_cli.dart

```diff
@@ -281,30 +284,68 @@ Future<int> status(ArgResults ar) async {
 }
 
 Future<AtClient> createAtClient(ArgResults ar) async {
+  final int maxConnectAttempts = 5;
   String nameSpace = 'at_activate';
   String atSign = AtUtils.fixAtSign(ar[AuthCliArgs.argNameAtSign]);
-  storageDir = standardAtClientStorageDir(
+  String? homeDir = HomeDirectoryUtil.homeDir;
+
+  storageDir = HomeDirectoryUtil.standardAtClientStorageDir(
     atSign: atSign,
     progName: nameSpace,
     uniqueID: '${DateTime.now().millisecondsSinceEpoch}',
   );
 
-  CLIBase cliBase = CLIBase(
-    atSign: atSign,
-    atKeysFilePath: ar[AuthCliArgs.argNameAtKeys],
-    nameSpace: nameSpace,
-    rootDomain: ar[AuthCliArgs.argNameAtDirectoryFqdn],
-    homeDir: getHomeDirectory(),
-    storageDir: storageDir!.path,
-    verbose: ar[AuthCliArgs.argNameVerbose] || ar[AuthCliArgs.argNameDebug],
-    syncDisabled: true,
-    maxConnectAttempts: int.parse(
-        ar[AuthCliArgs.argNameMaxConnectAttempts]), // 10 * 3 == 30 seconds
-  );
-
-  await cliBase.init();
+  // Get file path from the user. If null, search atKeys in the default filePath in user home directory.
+  String atKeysFilePathToUse = (ar[AuthCliArgs.argNameAtKeys] ??
+          '$homeDir/.atsign/keys/${atSign}_key.atKeys')
+      .replaceAll('/', Platform.pathSeparator);
+  String? localStoragePathToUse = storageDir?.path;
+  String? commitLogStoragePathToUse =
+      ('${storageDir?.path}/commit').replaceAll('/', Platform.pathSeparator);
+  String downloadPathToUse = ('$homeDir!/.atsign/downloads/$atSign/$nameSpace')
+      .replaceAll('/', Platform.pathSeparator);
 
```

Comment:
Can you extract the code below into its own file as a function which will return an AtClient given some args (similar to what we have in the noports repo [here](https://github.com/atsign-foundation/noports/blob/trunk/packages/dart/sshnoports/lib/src/create_at_client_cli.dart))